src/main/scala/com.snowplowanalytics.iglu/ctl/Main.scala

```diff
@@ -49,29 +51,32 @@ object Main extends IOApp {
 
   def run(args: List[String]): IO[ExitCode] = {
     val result: Result = Command.parse(args) match {
-      case Right(Command.Lint(input, skipChecks, skipSchemas)) =>
-        Lint.process(input, skipChecks, skipSchemas)
+      case Right(command: Command.Lint) =>
```

Comment:
While exploring if we could tidy up how the http client is created as suggested [here](https://github.com/snowplow-incubator/igluctl/pull/118#discussion_r903603323), I changed each `process` function to take the relevant `Command` or the relevant `Command` and an http client. This change isn't strictly necessary to move to http4s, but it does make things tidier, so I propose keeping those changes.